samples/api/hello_triangle/hello_triangle.cpp

```diff
@@ -896,12 +987,13 @@ void HelloTriangle::render_triangle(uint32_t swapchain_index)
  */
 VkResult HelloTriangle::present_image(uint32_t index)
 {
-	VkPresentInfoKHR present{VK_STRUCTURE_TYPE_PRESENT_INFO_KHR};
-	present.swapchainCount     = 1;
-	present.pSwapchains        = &context.swapchain;
-	present.pImageIndices      = &index;
-	present.waitSemaphoreCount = 1;
-	present.pWaitSemaphores    = &context.per_frame[index].swapchain_release_semaphore;
+	VkPresentInfoKHR present{
+	    .sType              = VK_STRUCTURE_TYPE_PRESENT_INFO_KHR,
+	    .waitSemaphoreCount = 1,
+	    .swapchainCount     = 1,
+	    .pSwapchains        = &context.swapchain,
+	    .pImageIndices      = &index};
+	present.pWaitSemaphores = &context.per_frame[index].swapchain_release_semaphore;
```

Comment:
Why do you keep the `pWaitSemaphores` out of the initializer list?

Reply:
That was unintentional.